### PR TITLE
Fix Drag pulse

### PR DESCRIPTION
**Pull request by rodolfocarobene**
Checklist before review:
- [x] Drag pulse with beta=0 produces same AllXY than gaussian
- [x] AllXY with beta caracterization gives sensible results

Checklist before merge:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibosoq/programs/base.py
Comment (AleCandido) [inline src/qibosoq/programs/base.py:130]:
```suggestion
                delta = - self.soccfg["gens"][gen_ch]["samps_per_clk"] * self.soccfg["gens"][gen_ch]["f_fabric"] / 2
```

It's just the same, but you avoid giving the same name to two different things.

**Review by AleCandido — APPROVED** [review]
Apart from the minor style comment, this should be a small technical PR.
So, if you are confident to be correct (and there is nothing I can tell about), it's good to go whenever you wish :)

**Comment by rodolfocarobene** [comment]
Thank you @AleCandido! Since this PR points to the other approved PR I'm merging it. I'll wait for the approval of #41 to merge all into main

**Comment by AleCandido** [comment]
> Thank you @AleCandido! Since this PR points to the other approved PR I'm merging it. I'll wait for the approval of #41 to merge all into main

Approved :D

**Pull request merged**
